bro-spinn/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/brospinn/App.kt

```diff
@@ -22,14 +22,14 @@ fun main() {
 }
 
 fun RapidsConnection.createEksternInntektsmeldingLoeser(spinnKlient: SpinnKlient): RapidsConnection =
-    apply {
-        logger.info("Starting ${EksternInntektsmeldingLoeser::class.simpleName}...")
+    also {
```

Comment:
Nei, jeg liker det fortsatt ikke.. "er lik også!?" ;)

Reply:
Er det ikke aller enklest å lese / forstå om man bare returnerer this :   
fun RapidsConnection.createEksternInntektsmeldingLoeser(spinnKlient: SpinnKlient): RapidsConnection {
    logger.info("Starting ${EksternInntektsmeldingLoeser::class.simpleName}...")
    EksternInntektsmeldingLoeser(this, spinnKlient)
    return this
}

Reply:
Det er nok subjektivt, men når du bruker `return this` så forventer jeg at du har mutert `this` på et vis.